### DISCUSSION TITLE
Site can't be reached

### DISCUSSION THREAD

**Discussion by ghost** [mozilla/hubs#5530] [Hubs Cloud Questions] [answered]
Hi,

I am Irin. 

We have our mozilla hubs cloud instance in the following url on AWS,

https://3d-ambasco.com/

We are in the process of building a room for an upcoming event. 

But our site is down. It has been down for the last 2 days. We need to continue working on the room tomorrow. We would like to know how we can get it back up running at the earliest. We are in need of urgent help. 

Also, I checked the status of the stack in cloud formation. It is online. The EC2 instance is also running. We are not sure what went wrong. 

Someone suggested to "Terminate instance" on AWS console. But we are hesitant to do it. We have the fear of losing all our work. Can rebooting help?

It would be great if this community can help us through this. Thank you.

Warm regards,

Irin Sajan

**Comment by rawnsley** [comment, marked as answer]
There is a problem with DNS for that domain, but it is hard to say where the problem originates. Hubs Cloud should setup your DNS entries in AWS Route 53, but it also relies on the registrar where you registered the domain pointing at the AWS name servers. This doesn't seem to be setup correctly: https://dnschecker.org/ns-lookup.php?query=3d-ambasco.com&dns=google

In this case AWS also seems to be the registrar: https://who.is/whois/3d-ambasco.com

You need to double check that the nameservers Route 53 requires for this domain are the same ones that the AWS Domains service is pointing at. Honestly it would be weird if they weren't, but that is the first thing to check.

**Reply by ghost** [reply]
Thank you for responding. It was domain trouble. The domain was taken down as email was not verified. Thank you for the help.
